apps/aecore/lib/aecore/chain/difficulty.ex

```diff
@@ -19,7 +23,9 @@ defmodule Aecore.Chain.Difficulty do
       distance = calculate_distance(list)
 
       next_difficulty =
-        (latest_block.header.difficulty_target * (@target_distance / distance))
```

Comment:
Does credo fail if it is done this way? @target_distance / distance can be separated in a variable to avoid using Kernel

Reply:
Credo requires when pipe operator is used to start with a single variable. Is using Kernel a problem here?

Reply:
Never mind, I thought it was failing because of the arithmetic expression.

Reply:
this look really weird

Reply:
Well, I could seperate it in another variable as @d-velev suggested

Reply:
yes okay